addons/hr_timesheet/report/project_report_view.xml

```diff
@@ -14,5 +14,18 @@
                  </graph>
              </field>
         </record>
+
+        <record id="view_task_project_user_pivot_inherited" model="ir.ui.view">
+            <field name="name">report.project.task.user.pivot.inherited</field>
+            <field name="model">report.project.task.user</field>
+            <field name="inherit_id" ref="project.view_task_project_user_pivot"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='project_id']" position='after'>
+                    <field name="hours_planned" widget="timesheet_uom"/>
+                    <field name="hours_effective" widget="timesheet_uom"/>
+                    <field name="remaining_hours" widget="timesheet_uom"/>
```

Comment:
For readability and comprehension to directly know those fields are measures and not `type="row"` 🙂 
```suggestion
                    <field name="hours_planned" widget="timesheet_uom" type="measure"/>
                    <field name="hours_effective" widget="timesheet_uom" type="measure"/>
                    <field name="remaining_hours" widget="timesheet_uom" type="measure"/>
```